core/embed/rust/src/ui/layout_delizia/component/address_details.rs

```diff
@@ -2,6 +2,7 @@ use heapless::Vec;
 
 use crate::{
     error::Error,
+    micropython::buffer::StrBuffer,
```

Comment:
Please also add `feature = "micropython"` gate to component/mod.rs so that it doesn't break non-micropython build. For this reason we generally avoid StrBuffer in `src/ui`, though here it makes sense as
* it's in an array,
* in a layout that's unlikely to be used by bootloader.

Reply:
Sounds good, thanks!
Fixed in aafe26b2dec3c65f286d1eae8a1d3c80ccdfcfd1.

Reply:
Also applied this optimization to Bolt and Caesar in f85b2565a7.